main.js

```diff
@@ -77,6 +86,21 @@ async function downloadCLI(version, platform, arch) {
   }
 }
 
+// Verifies the download attestation
+async function verifyDownloadAttestation(downloadPath, version) {
+  await exec.exec("gh", [
+    "attestation",
+    "verify",
+    downloadPath,
+    "--repo",
+    "UpCloudLtd/upcloud-cli",
+    "--signer-workflow",
+    "UpCloudLtd/upcloud-cli/.github/workflows/publish.yml",
```

Comment:
Mental note: this will break if the signer workflow filename changes (incl if it moves to a reusable one in another repo). I guess that's rather unlikely for now but understand if we'd rather leave it out.

Reply:
I guess this is something we can remove later if needed 🤔